clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -128,6 +128,11 @@ Improvements to clang-tidy
   as a value for `-export-fixes` to export individual yaml files for each
   compilation unit.
 
+- Improved :program:`clang-tidy-diff.py` script. It now accepts a `-config-file` option
```

Comment:
There is already entry for clang-tidy-diff.py, this could be appended there & maybe refactored to avoid constant "It now".

Reply:
Hope this is better now - unless nesting bullet points is not desirable.